src/decorators/debounce.ts

```diff
@@ -1,6 +1,12 @@
-import lodashDebounce, {DebounceSettings} from 'lodash/debounce';
+import debounce from 'lodash/debounce';
```

Comment:
so it looks like we can do 

```
import lodashDebounce from 'lodash/debounce';
import {DebounceSettings} from 'lodash';
```
to resolve this. `DebounceSettings` is a type definition and doesn't seems to increase the build file size.